Firestore/core/test/unit/bundle/bundle_reader_test.cc

```diff
@@ -379,11 +413,17 @@ TEST_F(BundleReaderTest, ReadsWithoutNamedQuery) {
       VerifyFullBundleParsed(reader, "bundle-1", testutil::Version(6000004000));
 
   EXPECT_EQ(elements.size(), 2);
-  VerifyDocumentMetadataEquals(
-      *reinterpret_cast<BundledDocumentMetadata*>(elements[0].get()),
-      DocumentMetadata1());
-  VerifyDocumentEncodesToOriginal(
-      *reinterpret_cast<BundleDocument*>(elements[1].get()), Document1());
+  {
+    SCOPED_TRACE("DocumentMetadata1");
```

Comment:
I think all `SCOPED_TRACE`s from here on can be removed.

Reply:
Done.